merge-eslint-reports.js

```diff
@@ -0,0 +1,11 @@
+const fs = require('fs');
+
+const directories = fs.readdirSync('./packages');
+const reports = directories.map((dir) => fs.readFileSync(`./packages/${dir}/report.json`, { encoding: 'utf-8' }));
+const merged = [];
+
+for (const report of reports) {
+  merged.push(...JSON.parse(report));
+}
```

Comment:
nit: use reduce to avoid array.push

Reply:
reduce is not clear in this case

Reply:
we allowed reduce in our lint config

Reply:
Yeah, I'm just nitpicking and trying to prevent you from merging this PR. :-)

Reply:
hahaha